doc/tech-docs/source/guides.html.md.erb

```diff
@@ -0,0 +1,50 @@
+---
+title: Guides
+source_url: 'https://github.com/ministryofjustice/hmpps-probation-integration-services/blob/main/doc/tech-docs/source/guides.html.md.erb'
+weight: 40
+---
+
+# How-To Guides
+
+## Making an Authenticated API Request
+
+Authentication tokens can be obtained from the internal HMPPS OAuth server,
+[HMPPS Auth](https://github.com/ministryofjustice/hmpps-auth), using Client
+Credentials Flow. Specific credentials for accessing an integration service
+are supplied by the 'HMPPS Auth, Audit and Registers' team. The client should
+be configured with the correct scopes and authorities for accessing the API
+endpoints provided by the integration service and these will then be present
+in the supplied JWT.
+
+```bash
+# Client credentials - supplied by 'HMPPS Auth, Audit and Registers' team
+CLIENT_ID=<hmpps-auth-client-id>
+CLIENT_SECRET=<hmpps-auth-client-secret>
+
+# MacOS - Generate basic auth. Default for base64 is no line wrapping
+BASIC_AUTH=$(echo -n $CLIENT_ID:$CLIENT_SECRET | base64)
+
+# Linux - Generate basic auth. Disable base64 line wrapping
+BASIC_AUTH=$(echo -n $CLIENT_ID:$CLIENT_SECRET | base64 -w 0)
+
+# Request a JWT access token
+AUTH_TOKEN=$(
+	curl -s --location \
+		--request POST "https://sign-in.hmpps.service.justice.gov.uk/auth/oauth/token?grant_type=client_credentials" \
+		--header "Authorization: Basic $BASIC_AUTH" |
```

Comment:
You could simplify this to:
```suggestion
		--user "$CLIENT_ID:$CLIENT_SECRET" |
```
then you don't need to worry about base64-encoding it yourself.

Reply:
Nice!